api/go.mod

```diff
@@ -15,9 +15,9 @@ require (
 	github.com/go-test/deep v1.0.2
 	github.com/hashicorp/errwrap v1.1.0
 	github.com/hashicorp/go-cleanhttp v0.5.2
-	github.com/hashicorp/go-hclog v0.16.2
+	github.com/hashicorp/go-hclog v1.6.3
```

Comment:
Is this version bump required? I am slightly worried about the major version bump causing issues here.